src/index.html

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="./esri-global-nav.css">
     <link rel="stylesheet" href="https://s3-us-west-1.amazonaws.com/patterns.esri.com/files/calcite-web/1.0.1/css/calcite-web.min.css">
```

Comment:
if you're pulling in fonts from the CDN, you should use the no-fonts version of calcite-web (and also bump up the version as well)